### PR TITLE
Drink type description

### PR DESCRIPTION
**Pull request by klaseskilson**
This add drink type descriptions and adds the brewery to the public drink list.

### CODE REVIEW

**Comment by tistatos** [comment]
I think it looks good! shouldn't this also be shown in the drink_types view as well?

File: app/views/drink_types/show.html.erb [hunk 1]
```diff
@@ -10,6 +10,7 @@
             </small>
           <% end %>
         </h1>
+        <%= content_tag :p, @drink_type.description, class: 'lead' unless @drink_type.description.blank? %>
```
Comment (klaseskilson) [inline app/views/drink_types/show.html.erb]: @tistatos it does!

**Comment by tistatos** [comment]
Sorry! My mistake, I confused this PR with #95 , well give it another spin tonite

**Comment by tistatos** [comment]
Perhaps add a cut-off for the description of its too long?

**Comment by klaseskilson** [comment]
I'll get to it!

**Comment by klaseskilson** [comment]
@tistatos 
screen shot 2016-02-15 at 22 46 23

**Pull request merged**